client/pages/saved-routes.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+export default class SavedRoutes extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      routes: null
+    };
+    this.handleClickRoute = this.handleClickRoute.bind(this);
+  }
+
+  componentDidMount() {
+    fetch('/api/routes')
+      .then(res => res.json())
+      .then(routes => {
+        this.setState({ routes });
+      });
+  }
+
+  handleClickRoute(event) {
+    const clickedRoute = event.target.closest('.route-list-item');
+    const routeId = clickedRoute.getAttribute('route-id');
+    this.props.selectRoute(routeId);
+  }
+
+  renderRoutes() {
+    if (!this.state.routes) {
+      return <p className="no-routes">No saved routes yet!</p>;
+    }
+    return (
+      this.state.routes.map(route => {
```

Comment:
It's good to include this message, but since arrays are always truthy (even empty arrays), Nobody will ever see it.